source/more-info/unsupported/container.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Containers known to cause issues"
+description: "More information on why certain containers marks the installation as unsupported."
+---
+
+## The issue
+
+The supervisor needs to be the only manager of the containers that it manages, unfortunately there are som other containers that also take on this role, and even worse they will auto-update the containers to development builds without consideration of the channel you have chosen, because of this a few containers known to cause issues have been added to a denylist that will flag your installation as unsupported.
```

Comment:
```suggestion
The Supervisor needs to be the only manager of the containers that it manages, unfortunately, there are some other containers that also take on this role. Worst case, those will auto-update the existing containers to development builds, without consideration of the channel you have chosen. Because of this, a few containers known to cause issues, have been added to a denylist that will flag your installation as unsupported.
```

Additionally, should we mention here that running additional software/container isn't supported at all?